### PR TITLE
Give Machinists medical records again

### PR DESCRIPTION
**Pull request by FlamingLily**
As the title says. Also included are some other science sort of stuff which they used to have, and seeing as they're still expected to do RnD judging by the RnD section of their workshop, I think it's not that radical.

### CODE REVIEW

**Comment by FlamingLily** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/modular_computers/file_system/programs/app_presets.dm [hunk 1]
```diff
@@ -513,6 +491,27 @@
 	)
 	return _prg_list
 
+/datum/modular_computer_app_presets/supply/robotics
+	name = "operations_robotics"
+	display_name = "Operations - Robotics"
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/file_system/programs/app_presets.dm:496]: This all needs renaming to machinist.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/modular_computers/file_system/programs/app_presets.dm (hunk 1 above)
Comment (FlamingLily) [reply code/modules/modular_computers/file_system/programs/app_presets.dm:496]: including paths or just the names?

**Review by FlamingLily — COMMENTED** [review]
changed names to machinist

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**